frontend/src/components/common/form/NotesModal.tsx

```diff
@@ -53,8 +54,10 @@ export const NotesModal: React.FunctionComponent<INotesModalProps> = ({
         okButtonText="Save"
         cancelButtonText="Cancel"
         handleOk={() => {
-          onSave && onSave(values);
-          setShowNotes(false);
+          if (!error) {
```

Comment:
this ensures that the notes modal does not allow a note to be saved if it exceeds the validation schema length.